sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -286,8 +284,8 @@ public void sync(CompletionListener listener) {
 
             if (throwable == null) {
                 Portal portal = getSafePortal(lastPortalName);
-                Plan plan = portal.prepareSync(planner);
-                applySessionSettings(plan);
+//FIXME: re-enable apply session settings functionality
```

Comment:
just remove it completely without comment, once we actually use the settings we'll re-add them as needed